packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React, { ReactElement, ReactNode, useCallback, useState } from "react";
+import { Tab as MuiTab } from "@mui/material";
+import {
+  TabList as MuiTabList,
+  TabPanel as MuiTabPanel,
+  TabContext as MuiTabContext,
+} from "@mui/lab";
+
+export type TabItemProps = {
+  children: ReactNode;
+  startIcon?: ReactElement;
+  label: string;
+  isDisabled?: boolean;
+  value?: string;
+};
+
+export type TabsProps = {
+  tabs: TabItemProps[];
+  initialValue?: string;
+  ariaLabel?: string;
+};
```

Comment:
Just noticed this. What's `initialValue`? You mean the initially selected tab?

And by default we're using string indexes, so this works right?

Reply:
Is there no way to manually change the value of the tabs or listen for changes to which tab is selected? I'm pretty sure we need that in some places. Not confirmed, but I can see someone asking for that.

If you'd like, we can do that in a fast-follow; just make a ticket.